src/app/api/request-schedule/route.ts

```diff
@@ -0,0 +1,44 @@
+import { prisma } from "@/lib/prisma";
+import { NextResponse } from "next/server";
+
+export async function GET() {
```

Comment:
This is just a comment - no need to change code - but we should discuss.
I'm thinking this refers to step 1 in the algo? Do we really want to send this to the client? I was thinking that calculation is all back-end and we use it to come up with an input to the google routes API request?